resources/TrialApp.didact.md

```diff
@@ -11,13 +11,15 @@ This tutorial will show you how to make and deploy changes to your Azure App Ser
 ![Trial app view](./trialAppTutorial/viewTrialAppScreenshot.png)
 
 2. Right-click on the trial app.
-3. Select `Clone`.
+3. Select `Clone Trial App...`.
```

Comment:
I think you should get rid of the period at the end here.  It looks weird preceded with `...`